operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -3810,6 +3810,129 @@
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  }
+                ]
+              },
+              "unit": "none"
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 6,
+            "w": 3,
+            "x": 0,
+            "y": 67
+          },
+          "id": 476,
+          "options": {
+            "colorMode": "value",
+            "graphMode": "area",
+            "justifyMode": "auto",
+            "orientation": "auto",
+            "reduceOptions": {
+              "calcs": [
+                "lastNotNull"
+              ],
+              "fields": "",
+              "values": false
+            },
+            "text": {},
+            "textMode": "auto"
+          },
+          "pluginVersion": "9.5.17",
+          "targets": [
+            {
+              "editorMode": "code",
+              "exemplar": true,
+              "expr": "sum(watcher_workqueue_depth{container='tekton-chains-controller',app='tekton-chains-controller'})",
+              "interval": "",
+              "intervalFactor": 1,
+              "legendFormat": "Queue Depth",
+              "range": true,
+              "refId": "work queue depth"
+            }
+          ],
+          "title": "Tekton Chains Work Queue Depth",
+          "type": "stat"
+        },
+        {
+          "description": "Reconcile latency 99%-tile - \nLatency of reconcile operations.",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "fixedColor": "green",
+                "mode": "thresholds"
+              },
+              "mappings": [],
+              "noValue": "No Data",
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  }
+                ]
+              },
+              "unit": "ms"
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 6,
+            "w": 4,
+            "x": 3,
+            "y": 67
+          },
+          "id": 477,
+          "interval": "15s",
+          "options": {
+            "colorMode": "value",
+            "graphMode": "area",
+            "justifyMode": "auto",
+            "orientation": "auto",
+            "reduceOptions": {
+              "calcs": [
+                "lastNotNull"
+              ],
+              "fields": "",
+              "values": false
+            },
+            "text": {},
+            "textMode": "auto"
+          },
+          "pluginVersion": "9.5.17",
+          "targets": [
+            {
+              "editorMode": "code",
+              "exemplar": true,
+              "expr": "histogram_quantile(0.99, sum(rate(watcher_reconcile_latency_bucket{job=\"tekton-chains\"}[30m])) by (le) ) / 1000",
```

Comment:
I had to convert `\"` to just `"` @jkhelil but `histogram_quantile(0.99, sum(rate(watcher_reconcile_latency_bucket{job="tekton-chains"}[30m])) by (le) ) / 1000` also works from the OCP metrics console ... see if you can do it as well

Reply:
it is just a way to escape doublequote inside the doublequote, it is used all over the json file